<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>626</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simulation Ensemble Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QStackedWidget" name="samplePages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="chooseSchemePage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="samplingTabs">
         <property name="toolTip">
          <string>Identify the input parameters with uncertainty and characterize their probability distributions.</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="distTab">
          <attribute name="title">
           <string>Distributions</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="SimSetupInputsTable" name="distTable">
             <property name="maximumSize">
              <size>
               <width>100000</width>
               <height>1000000</height>
              </size>
             </property>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Default</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Min</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Max</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>PDF</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Param 1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Param 2</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="allFixedButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Click to set all input parameters to fixed.</string>
               </property>
               <property name="text">
                <string>All Fixed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="allVariableButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Click to set all input parameters to variable.</string>
               </property>
               <property name="text">
                <string>All Variable</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Sampling scheme</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Different sampling schemes are required to achieve certain UQ tasks.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Show schemes:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="allSchemesRadio">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="paramScreenRadio">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>For parameter screening only</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="otherRadio">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>For response surface analysis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="adaptiveRefineRadio">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>For adaptive response surface analysis</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QListWidget" name="schemesList">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Select the sampling scheme suitable for your analysis.
Each sampling scheme will fill the space differently.
You can always preview your ensemble after it is generated.</string>
             </property>
             <item>
              <property name="text">
               <string>Monte Carlo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Quasi Monte Carlo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>METIS</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string># of samples?</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSpinBox" name="numSamplesBox">
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="generateSamplesButton">
                 <property name="text">
                  <string>Generate Samples</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="generateStatusText">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadSamplesPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Load samples</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Sample</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>501</width>
            <height>33</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="sampleFileNameEdit">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="adaptiveFSDataPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="dataViewWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>143</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="previewButton">
     <property name="text">
      <string>Preview Samples</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="doneButton">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>95</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Choose how to generate samples:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="chooseSchemeRadio">
        <property name="toolTip">
         <string>Choose this option if you want to generate
a new ensemble.</string>
        </property>
        <property name="text">
         <string>Choose sampling scheme</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="flowsheetDataRadio">
        <property name="toolTip">
         <string>Choose this option if you have an existing
ensemble from your flowsheet.</string>
        </property>
        <property name="text">
         <string>Load flowsheet samples</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="loadSamplesRadio">
        <property name="toolTip">
         <string>Choose this option if you have an existing
ensemble saved as a PSUADE file.</string>
        </property>
        <property name="text">
         <string>Load all samples from a single file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimSetupInputsTable</class>
   <extends>QTableWidget</extends>
   <header>foqus_lib/gui/uq/SimSetupInputsTable</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doneButton</tabstop>
  <tabstop>previewButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>samplingTabs</tabstop>
  <tabstop>distTable</tabstop>
  <tabstop>allFixedButton</tabstop>
  <tabstop>allVariableButton</tabstop>
  <tabstop>sampleFileNameEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>allSchemesRadio</tabstop>
  <tabstop>paramScreenRadio</tabstop>
  <tabstop>otherRadio</tabstop>
  <tabstop>adaptiveRefineRadio</tabstop>
  <tabstop>schemesList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
